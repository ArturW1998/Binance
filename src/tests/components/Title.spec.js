import React from 'react';
import { shallow } from 'enzyme';
import Title from '../../components/Title';

describe('Title component', () => {
  describe('Title component with title', () => {
    const props = {
      title: "The World's Leading Cryptocurrency Exchange",
    };

    const title = shallow(<Title {...props} />);

    it('renders properly', () => {
      expect(title).toMatchSnapshot();
    });

    it('renders <h2>', () => {
      expect(title.find('h2')).toHaveLength(1);
    });

    it('renders title text', () => {
      expect(title.find('h2').text()).toEqual(
        "The World's Leading Cryptocurrency Exchange",
      );
    });
  });

  describe('Title component with default props', () => {
    const props = {};

    const title = shallow(<Title {...props} />);

    it('renders properly', () => {
      expect(title).toMatchSnapshot();
    });

    it('renders title text', () => {
      expect(title.find('h2').text()).toEqual('Simple title');
    });
  });
});
