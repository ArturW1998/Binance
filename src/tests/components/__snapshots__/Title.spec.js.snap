// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component Title component with default props renders properly 1`] = `ShallowWrapper {}`;

exports[`Title component Title component with title renders properly 1`] = `ShallowWrapper {}`;
